html/php-pre-challenge2/index.php

```diff
@@ -1,12 +1,36 @@
 <?php
-$array = explode(',', $_GET['array']);
-
-// 修正はここから
-for ($i = 0; $i < count($array); $i++) {
+$array = explode(',', $_GET['array']);//Getで受け取った配列を変数に格納する
+echo'ソート前';
+echo "<pre>";
+print_r($array);
+echo "</pre>";
 
+ /**
+  * クイックソートする
+  *@param 配列 $array
+  *@return 配列 $result
```

Comment:
Docコメントはphpが判別できる型を書きましょう
```suggestion
 * @param array $array
 * @return array $result
```

Reply:
f56c12856063dddaaa7bfbec901ee1ec5642cf49
このコミットで対応しました！